src/olympia/ratings/views.py

```diff
@@ -389,6 +393,14 @@ def filter_queryset(self, qs):
                 # because the frontend wants to call this before and after
                 # having posted a new rating, and needs accurate results.
                 self.pagination_class = OneOrZeroPageNumberPagination
+            if score_filter:
+                try:
+                    scores = [int(score) for score in score_filter.split(',')]
+                except ValueError:
```

Comment:
You may need to handle non-ascii input, too

```
Python 2.7.10 (default, Feb  7 2017, 00:08:15)
>>> int(u'Krstić')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'decimal' codec can't encode character u'\u0107' in position 5: invalid decimal Unicode string
```

Reply:
They're from `request.GET` so the input should already be encoded as a byte string.  It's easily testable - it's the same method that `exclude_ratings` uses parses the input so if you can't 500 on `exclude_ratings=Krstić` then it's fine.

Reply:
Actually it does - It's just that `UnicodeEncodeError` - which is raised here - is inheriting from `ValueError`.